### ISSUE TITLE
Add project not available page and loader to edit project

### ISSUE THREAD

**Issue by RamRamez** [Giveth/giveth-dapps-v2#1079]
1. add a loader to edit project
2. first check if user is signed in then check if user is registered. If user is not signed in, complete profile modal shouldn't be shown.
3. show project not available page in edit project if project not found
4. auto close welcome modal when user singed in, in edit project view

**Comment by MoeNick** [comment]
@divine-comedian

**Comment by divine-comedian** [comment]
1. loader shows successfully on edit project
2. this isn't clear on what page I should be on @RamRamez  - the homepage or edit project view?
3. I successfully see the error page visiting a project id that doesn't exist

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/67759413/184925210-bbd10b7f-8496-4f47-9f1e-c3e0607f5c4f.png">

4. sign-in modal automatically closes when I sign in with metamask. this looks good!

### one small bug I found
if I am not signed in and I go to the edit project URL I see the sign in modal, if I close the sign in modal I see the infinite loading wheel for edit project
https://www.loom.com/share/be331fe75c9b47ff83c7de2c77435efd

user should be informed they need to sign in or return to homepage

**Comment by mosaeedi** [comment]
Thanks for raising the issue!

Here is the design for the not signed-in state.

[Figma link](https://www.figma.com/file/nVoinu0tgJ565enN5R4WDE/Giveth.io-%26-GIVeconomy?node-id=13439%3A242788)

**Comment by RamRamez** [comment]
> Thanks for raising the issue!
> 
> Here is the design for the not signed-in state.
> 
> [Figma link](https://www.figma.com/file/nVoinu0tgJ565enN5R4WDE/Giveth.io-%26-GIVeconomy?node-id=13439%3A242788)
> 
> And the design for not connected wallet state! [Figma link](https://www.figma.com/file/nVoinu0tgJ565enN5R4WDE/Giveth.io-%26-GIVeconomy?node-id=13439%3A243454)

This deign is implemented in PR #1482 

For testing:
I added `wallet not connected` and `user not signed in` design to:
1. Verification
2. Onboarding
3. create and edit project
4. account view

I found a bug and fixed it in this PR
In some pages (like ) there is no footer. the bug is when user return to another page (that has footer) from that page, there's no footer in the new page. I fixed it in these pages:
1. Verification
5. Verification - email Verification page
6. Create and edit project page
7. success page (after create publishing a project)

Also loading component is improved in edit and create project pages.
